pkgs/misc/uboot/default.nix

```diff
@@ -374,7 +374,10 @@ in {
 
   ubootOlimexA64Teres1 = buildUBoot {
     defconfig = "teres_i_defconfig";
-    extraMeta.platforms = ["aarch64-linux"];
+    extraMeta = {
+      platforms = ["aarch64-linux"];
+      maintainers = with maintainers; [ kreyren ];
```

Comment:
Hmm, this overrides the default maintainers rather than adding, which probably isn't right?

Reply:
Sorry! Forgot that the update operator takes the latter value if the same attribute name is present.

Adjusted in https://github.com/NixOS/nixpkgs/pull/309589/commits/1f4912f61a8009db00ad203b3a5fbb62232b252d and checking with `nix eval --json '.#PKGNAME.meta' | jq .` seems to work as expected